dirty_cat/_datetime_encoder.py

```diff
@@ -64,10 +64,10 @@ class DatetimeEncoder(BaseEstimator, TransformerMixin):
         Number of features in the data seen during fit.
     n_features_out_: int
         Number of features of the transformed data.
-    features_per_column_: typing.Dict[int, typing.List[str]]
+    features_per_column_: mapping of int to list of str
```

Comment:
I have the impression that we are always changing those typing documentation. This is making review hard, and I fail to see where the benefit is for the user.